sdk/loadtestservice/load-testing-rest/samples-dev/sample.ts

```diff
@@ -77,9 +93,17 @@ async function main() {
     },
   });
 
+  if (testRunCreationResult.status !== "200") {
+    throw testRunCreationResult.body.error;
+  }
+
   // Checking the test run status and printing metrics
-  var result = await client.path("/testruns/{testRunId}", testRunId).get();
-  console.log(result);
+  const getTestRunResult: any = await client.path("/testruns/{testRunId}", testRunId).get();
```

Comment:
We're not showing what users can do with the response of the test results? What kind of information does the service return? How will users use it?

Reply:
The API here returns test run results with related results and metrics, user can then use this data to analyze the test results.